src/Form/FormAutosuggest.jsx

```diff
@@ -157,6 +157,15 @@ function FormAutosuggest({
     };
   });
 
+  useEffect(() => {
+    if (value) {
```

Comment:
Should we also check that `value !== state.displayValue` to avoid unnecessary re-renders? I think without it if I open a dropdown and manually select an item it will first update `displayValue` internally and then call `onSelected` function that will update `value` state of a parent component that will trigger `setState` in this `useEffect` which is not needed because `displayValue` is already up-to-date.

Also, should we consider a case when `value` becomes an empty string for some reason? I think we might want to update `displayValue` in such case too, just in case.

Reply:
@viktorrusakov  thanks for the suggestion. 

Also regarding your second point, if `value` changes to `''` from lets say some value `test value`, then i guess this useEffect will run and will update the value to `''`. So this will also work.

Reply:
Yeah, that's true. Thanks for the update, LGTM!